### ISSUE TITLE
changing the category of a frame corrupts the sf_category tag

### ISSUE THREAD

**Issue by varioustoxins** [uwbmrb/PyNMRSTAR#117]
in `def category(self, category):` there appears to be an error

```python
# Update the sf_category tag too
lc_tags = self._lc_tags
if 'sf_category' in lc_tags:
    tag_index = lc_tags['sf_category']
    self.tags[tag_index] = ['sf_category',category]
```

when run with

```
   save_nef_nmr_spectrum_k_ubi_hnco`1`

      _nef_nmr_spectrum.sf_category                   nef_nmr_spectrum
      _nef_nmr_spectrum.sf_framecode                  nef_nmr_spectrum_k_ubi_hnco`1`
```

when we print the frame I get

```
save_nef_test_k_ubi_hnco`1`
   _nef_nmr_spectrum.n                                     e
   _nef_nmr_spectrum.sf_framecode                 nef_test_k_ubi_hnco`1`

```

the correct code should be 

```python
# Update the sf_category tag too
lc_tags = self._lc_tags
if 'sf_category' in lc_tags:
    tag_index = lc_tags['sf_category']
    self.tags[tag_index] = ['sf_category',category]
```

**Comment by jonwedell** [comment]
Thanks for the report! Bug confirmed, and fixed in 82f435a. I'll close this issue once I've got a new release available on PyPI - hopefully within the next 48 hours.

**Comment by jonwedell** [comment]
3.3.2 deployed to PyPI.
